### PR TITLE
Update synopsys references to blackduck

### PR DESCRIPTION
**Pull request by devmehtasynopsys**
Update synopsys references to blackduck. This PR contains all the changes required for sierra migration.

- Copyrights
- Group Names
- Plugin Display Name
- Import Statements
- Jenkins UI Changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Overview
-Synopsys Detect for Jenkins simplifies running Synopsys Detect in your Jenkins builds.
+Detect for Jenkins simplifies running Detect in your Jenkins builds.
```
Comment (gopannabd) [inline README.md:2]: Should it be Black Duck Detect here like it is elsewhere?

**Review by gopannabd — CHANGES_REQUESTED** [review]
Where we have just Detect instead of Synopsys Detect in couple of places, we should use Black Duck Detect.

**Review by gopannabd — APPROVED** [review]

**Review by niravrsynopsys — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Overview
-Synopsys Detect for Jenkins simplifies running Synopsys Detect in your Jenkins builds.
+Black Duck Detect for Jenkins simplifies running Detect in your Jenkins builds.
 
 # Build
-[![Build Status](https://travis-ci.org/jenkinsci/synopsys-detect-plugin.svg?branch=master)](https://travis-ci.org/jenkinsci/synopsys-detect-plugin)
+[![Build Status](https://travis-ci.org/jenkinsci/blackduck-detect-plugin.svg?branch=master)](https://travis-ci.org/jenkinsci/blackduck-detect-plugin)
```
Comment (dmamidisynopsys) [inline README.md:5]: @devmehtasynopsys Lets just remove this all together (travis link). This is not operational

**Review by dmamidisynopsys — COMMENTED** [review]
Changes..

File: README.md (hunk 2 above)
Comment (devmehtasynopsys) [reply README.md]: Removed

**Review by dmamidisynopsys — APPROVED** [review]

**Pull request merged**